### PR TITLE
Update issue template for support bundle

### PR DESCRIPTION
**Pull request by andrewtchin**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -28,10 +28,9 @@ REQUIRED:
 - Does the VIC appliance recieve configuration by DHCP?
 - What stage of the Appliance Lifecycle is the VIC appliance in?
   https://github.com/vmware/vic-product/blob/master/installer/docs/SUPPORT.md#appliance-lifecycle
-- Attach the VIC appliance support bundle
+- Attach the VIC appliance support bundle (e.g. vic_appliance_logs_2018-01-01-00-01-00.tar.gz)
   https://github.com/vmware/vic-product/blob/master/installer/docs/SUPPORT.md#appliance-support-bundle
-  If you are not comfortable with posting this publicly, please email it to andrewchin@vmware.com
-  and reference the created issue number in the subject
+  If you are not comfortable with posting this publicly, please email it to andrewchin@vmware.com and reference the created issue number in the subject
```
Comment (morris-jason) [inline .github/ISSUE_TEMPLATE.md]: Is there another email we can use? Do we have a vic-eng email?

File: .github/ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -28,10 +28,9 @@ REQUIRED:
 - Does the VIC appliance recieve configuration by DHCP?
 - What stage of the Appliance Lifecycle is the VIC appliance in?
   https://github.com/vmware/vic-product/blob/master/installer/docs/SUPPORT.md#appliance-lifecycle
-- Attach the VIC appliance support bundle
+- Attach the VIC appliance support bundle (e.g. vic_appliance_logs_2018-01-01-00-01-00.tar.gz)
```
Comment (morris-jason) [inline .github/ISSUE_TEMPLATE.md]: Should we call out that customers probably shouldn't upload the private bundle?

File: .github/ISSUE_TEMPLATE.md (hunk 1 above)
Comment (andrewtchin) [reply .github/ISSUE_TEMPLATE.md]: i was going to ask matt to see

File: .github/ISSUE_TEMPLATE.md (hunk 2 above)
Comment (andrewtchin) [reply .github/ISSUE_TEMPLATE.md]: ill add a warning

**Pull request merged**